apps/dashboard/src/components/contract-functions/contract-function.tsx

```diff
@@ -253,6 +262,23 @@ type ExtensionFunctions = {
   functions: AbiFunction[];
 };
 
+const FunctionInputSearch = ({
+  searchFn,
+}: {
+  searchFn: DebouncedState<(e: ChangeEvent<HTMLInputElement>) => void>;
+}) => (
+  <div className="sticky top-0 z-10 mb-3">
```

Comment:
try and keep the z-index as low as possible, lets use z-1 here instead ( please verify that it works though everywhere )